sn_evm/CHANGELOG.md

```diff
@@ -266,7 +266,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [0.14.35](https://github.com/maidsafe/safe_network/compare/sn_transfers-v0.14.34...sn_transfers-v0.14.35) - 2024-01-09
 
 ### Added
-- *(client)* extra sleep between chunk verification
+- *(client)* extra sleep between data verification
 
```

Comment:
Not sure we should change this in the Changelog file too!!

Reply:
ups! FNR gone TOO FAR